simulator/src/main/java/com/hazelcast/simulator/coordinator/deployment/AgentWorkerLayout.java

```diff
@@ -46,27 +44,23 @@
         this.agentWorkerMode = agentWorkerMode;
     }
 
-    public List<WorkerProcessSettings> getWorkerProcessSettings() {
```

Comment:
Why do these methods need to be public? Are they accessed outside of the package?